MojioSDK/Client/Core/RestClient.swift

```diff
@@ -360,11 +369,24 @@ open class RestClient {
             },
             encoding: URLEncoding(destination: .methodDependent),
             headers: self.defaultHeaders)
-            .responseData(queue: self.dispatchQueue) {response in
-                self.handleResponse(response, completion: completion, failure: failure)
-        }
-        
+            
+            
+        let debugDesc = request.debugDescription
         print(request)
+
+        request.responseData(queue: self.dispatchQueue) {response in
+            
+            // PHIOS-5207: post request notification for any loggers
+            let duration = response.timeline.requestDuration
+            let timestamp = Date()
+            let debugObj = RestClientRequestDebugInfo(urlString: self.requestUrl!,
+                                                      cURLRepresentation: debugDesc,
+                                                      duration: duration,
+                                                      timestamp: timestamp)
+            NotificationCenter.default.post(name: RestClientRequestNotificationName, object: debugObj)
```

Comment:
@davidjed  Do we have any triggers to switch on/off this feature? How it will impact Force app?

Reply:
@davidjed  Why do we cover only one kind of requests? What about `runStringBody`, `runEncodeJSON`, `runEncodeUrl`? I would prefer to implement this notification on the app level.

Reply:
There's a trigger in the app code based on ten taps in the environment switcher screen (similar to Android); I was thinking we could make that propagate to this layer to only fire the notification when triggered.

Reply:
@alex-kepar good catch on other run methods. Yes, should add this to there as a common call. I was thinking of putting it in the app level but I wanted to capture the CURL representation (often requested by platform) and Alamofire used in RestClient provides that.

Reply:
@davidjed  I meant trigger for switching off this behavior at all (do not send notifications into Notification center) if main app is not interested in it. For example we can do it in "RxRestClient" extension which overlaps RestClient in the main app.

Reply:
Going to try that if I can get appropriate debug data.

Reply:
OK, I think I can implement in RxRestClient and avoid this whole NotificationCenter issue by simply including the cURL string in the response notification. Will still require a small SDK change but it'll be relatively minor and won't require any notifications fired from SDK. Will resubmit this with this change. Thanks @alex-kepar

Reply:
@davidjed completely agree with @alex-kepar so would be better to implement it on the particular app side.